ppl/src/test/java/org/opensearch/sql/ppl/calcite/CalcitePPLMathFunctionTest.java

```diff
@@ -44,20 +44,37 @@ public void testAbsWithOverriding() {
   }
 
   @Test
-  public void testSqrtWithOverriding() {
-    String ppl = "source=EMP | eval SQRT = sqrt(4) | head 2 | fields SQRT";
+  public void testAcosWithOverriding() {
+    String ppl = "source=EMP | eval ACOS = acos(0) | head 2 | fields ACOS";
     RelNode root = getRelNode(ppl);
     String expectedLogical =
-        "LogicalProject(SQRT=[$8])\n"
+        "LogicalProject(ACOS=[$8])\n"
             + "  LogicalSort(fetch=[2])\n"
             + "    LogicalProject(EMPNO=[$0], ENAME=[$1], JOB=[$2], MGR=[$3], HIREDATE=[$4],"
-            + " SAL=[$5], COMM=[$6], DEPTNO=[$7], SQRT=[SQRT(4)])\n"
+            + " SAL=[$5], COMM=[$6], DEPTNO=[$7], ACOS=[ACOS(0)])\n"
             + "      LogicalTableScan(table=[[scott, EMP]])\n";
     verifyLogical(root, expectedLogical);
-    String expectedResult = "SQRT=2.0\n" + "SQRT=2.0\n";
+    String expectedResult = "ACOS=1.5707963267948966\n" + "ACOS=1.5707963267948966\n";
     verifyResult(root, expectedResult);
+    String expectedSparkSql = "SELECT ACOS(0) `ACOS`\n" + "FROM `scott`.`EMP`\n" + "LIMIT 2";
+    verifyPPLToSparkSQL(root, expectedSparkSql);
+  }
 
-    String expectedSparkSql = "" + "SELECT SQRT(4) `SQRT`\n" + "FROM `scott`.`EMP`\n" + "LIMIT 2";
+  @Test
+  public void testAsinWithOverriding() {
+    String ppl = "source=EMP | eval ASIN = asin(0) | head 2 | fields ASIN";
+    RelNode root = getRelNode(ppl);
+    String expectedLogical =
+        "LogicalProject(ASIN=[$8])\n"
+            + "  LogicalSort(fetch=[2])\n"
+            + "    LogicalProject(EMPNO=[$0], ENAME=[$1], JOB=[$2], MGR=[$3], HIREDATE=[$4],"
+            + " SAL=[$5], COMM=[$6], DEPTNO=[$7], ASIN=[ASIN(0)])\n"
+            + "      LogicalTableScan(table=[[scott, EMP]])\n";
```

Comment:
We should verify the precise content rather than the entire query plan. For example, if LogicalSort does not appear in the query plan, all irrelevant unit tests must be updated accordingly.

Reply:
yes, @yuancu please simply the UTs and complicate the ITs